EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFSetup.cc

```diff
@@ -22,39 +22,14 @@ namespace l1t {
         std::array<int, 12> board_out = {{1, 7, 2, 8, 3, 9, 4, 10, 5, 11, 6, 12}};  //these are board_ids per amc_no-1
 
         for (unsigned int i = 1; i <= board_out.size(); i++) {
```

Comment:
consider using range-based loop.

Reply:
same here, please give me feedback after the changes